subiquity/server/controllers/install.py

```diff
@@ -597,6 +597,11 @@ async def create_core_boot_classic_fstab(self, *, context):
         with open(self.tpath("etc/fstab"), "w") as fp:
             fp.write("/run/mnt/ubuntu-boot/EFI/ubuntu /boot/grub none bind\n")
 
+    @with_context(description="installing packages to live system")
+    async def install_live_packages(self, *, context):
```

Comment:
This will show "installing packages to live system" to the user even if there are no packages being installed. It's annoying to fix but that's kind of bad UX.

Reply:
good point, fixed